homeassistant/components/neptune_apex/strings.json

```diff
@@ -0,0 +1,24 @@
+{
+  "title": "Neptune Apex",
+  "config": {
+    "step": {
+      "user": {
+        "title": "Connect to Neptune Systems Apex Aquacontroller",
+        "description": "",
```

Comment:
We don't need to specify description if there's none.

Reply:
OK